aten/src/ATen/native/quantized/cpu/qnnpack/src/conv-prepack.cc

```diff
@@ -22,7 +22,7 @@ PrePackConvWeights::PrePackConvWeights(
       const uint32_t c_stride = (groups + (cr - 1)) & -cr;
       const size_t packed_weights_size =
           (sizeof(uint8_t) * kernel_size + sizeof(int32_t)) * c_stride;
-      packed_weights_ = aligned_alloc(16, packed_weights_size);
+      packed_weights_ = malloc(packed_weights_size);
```

Comment:
Is this needed for this to work or is this some sort of optimization?

Reply:
This was an optimization. It was originally using `malloc` in QNNPACK. But the android build fails with `aligned_alloc` so I switched back to `malloc`